libraries/AP_ExternalAHRS/AP_ExternalAHRS.cpp

```diff
@@ -334,6 +335,7 @@ void AP_ExternalAHRS::update(void)
                                     state.location.lat, state.location.lng, state.location.alt*0.01,
                                     filterStatus.value);
     }
+#endif
```

Comment:
technically we like this:

```suggestion
#endif  // HAL_LOGGING_ENABLED
```

once the excluded block gets lengthy.  But meh.